hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -370,11 +370,13 @@ int OtbnModel::check() const {
 
   good &= OtbnTraceChecker::get().Finish();
 
-  try {
-    good &= check_dmem(*iss);
-  } catch (const std::exception &err) {
-    std::cerr << "Failed to check DMEM: " << err.what() << "\n";
-    return -1;
+  if (iss->get_mirrored().wipe_start) {
```

Comment:
Can you add a comment here explaining why we only do it at this point? It's probably going to look a bit mysterious when we come back to the code in a few months...